### PR TITLE
Fix #1311: File Manager Page Styling

### LINKED ISSUE #1311: File Manager Tab Needs Restyling
Currently, the File Manager tab is poorly formatted and has an abrasive gray background.

image

### PR DESCRIPTION
**Pull request by jonathanzhang53**
Fixes #1311 

Fixed the abrasive colors on the file manager page and did some small changes to the UI.

See before and after below:
image
image

### CODE REVIEW

File: sites/default/filemanager/scan.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ function scan($dir){
 				);
 			}
 		}
-	
+
```
[the remark this answers is not in the document]
Comment (jonathanzhang53) [reply sites/default/filemanager/scan.php]: Sorry about these. I did not purposely do these removals. They were automatically done by some checking system, not sure what though. Should I still remove them?

**Comment by jonathanzhang53** [comment]
@muarachmann Do you still want me to fix the whitespace or following a certain styling guideline. Those blank lines were already there, my PR just removed the spaces that were on the blank lines.

**Comment by Trodrige** [comment]
Good job @jonathanzhang53 
Just remove the trailing before I can approve this, else @muarachmann will hit my head with a hammer.😉

**Comment by jonathanzhang53** [comment]
@Trodrige The code came like this. The trailings were there but some of them had actual spaces/tabs on them. My PR just got rid of those spaces, am I correct to assume that I need to go through and eliminate all the trails?

File: sites/default/filemanager/scan.php [hunk 2]
```diff
@@ -16,9 +16,9 @@ function scan($dir){
 	// Is there actually such a folder/file?
 
 	if(file_exists($dir)){
-	
+
 		foreach(scandir($dir) as $f) {
-		
+
```
[the remark this answers is not in the document]
Comment (jonathanzhang53) [reply sites/default/filemanager/scan.php]: There should be a blank line here to match with rest of code.

**Comment by aethelwulffe** [comment]
Mind you, I like the changes and am giving an initial approval waiting on another review.

**Comment by aethelwulffe** [comment]
No second review, and this is getting old.  As it is my code being changed, I will approve and merge as there has been no protest to it in the last three months!

**Pull request merged**